### ISSUE TITLE
fatal: ambiguous argument 'master..HEAD'

### ISSUE THREAD

**Issue by eisraeli** [cloudflare/pint#559]
When running `pint ci --base-branch=main` I get the following:
`level=info msg="Loading configuration file" path=.pint.hcl
level=fatal msg="Execution completed with error(s)" error="failed to get the list of commits to scan: fatal: ambiguous argument 'master..HEAD': unknown revision or path not in the working tree.\nUse '--' to separate paths from revisions, like this:\n'git <command> [<revision>...] -- [<file>...]'\n"`

Am I missing something here?

**Comment by prymitive** [comment]
Fixed in v0.42.1

**Comment by QSummerY** [comment]
I try to curl release in Jenkins, and then execute the pint ci command, but the same problem occurs. The release version used is 0.43.1. Do I need to configure something additionally?
```
./pint-linux-386 ci --base-branch main
level=info msg="Loading configuration file" path=.pint.hcl
level=fatal msg="Execution completed with error(s)" error="failed to get the list of commits to scan: fatal: ambiguous argument 'main..HEAD': unknown revision or path not in the working tree.\nUse '--' to separate paths from revisions, like this:\n'git <command> [<revision>...] -- [<file>...]'\n"
```
